en/ros/external_position_estimation.md

```diff
@@ -119,17 +225,19 @@ At this point, if you followed those steps, you are ready to test your setup.
 
```

Comment:
@TSC21 is this first flight stuff relevant to both vision and mocap? Some of it is specific currently "Put the robot on the ground and start streaming MoCap feedback. "

Reply:
Using vision egomotion and using mocap are two completely different approaches. So this should be under something specific to MoCap. To vision, there are other concerns as well.

Reply:
Can you provide information about what concerns are relevant to vision? I will move this section.